ansible/roles/containerd/README.adoc

```diff
@@ -0,0 +1,52 @@
+== Role: Kubernetes Cluster
+
+Install containerd on host.
+
+=== Description
+
+This role will install a containerd on a linux machine.
+
+
+=== Requirements
+
+TODO
```

Comment:
Can you add the requirements if they exist and remove TODO when done ?

Reply:
Removed the TODO from the requirements.

Reply:
Removed the TODO and also improved the document itself.